src/logging.cpp

```diff
@@ -239,13 +239,86 @@ namespace BCLog {
     }
 }
 
-void BCLog::Logger::LogPrintStr(const std::string& str, const std::string& logging_function, const std::string& source_file, const int source_line)
+static constexpr uint64_t HOURLY_LOG_QUOTA_IN_BYTES_PER_SOURCE_LOCATION{1024 * 1024};
+
```

Comment:
Comment needed describing what this function does at a high level and documenting the return value.

Reply:
There is a comment in `src/logging.h`.